backend/app/services/project_device_session_service.py

```
@@ -31,17 +31,17 @@ async def start_project_device_session(
-    project = (
-        db.query(Project)
-        .filter(
-            Project.id == project_id,
-            Project.user_id == user_id,
-            Project.is_active == True,
-        )
-        .first()
+    query = db.query(Project).filter(
+        Project.id == project_id,
+        Project.user_id == user_id,
+        Project.is_active == True,
```

Ruff (0.15.14)

[error] Avoid equality comparisons to `True`; use `Project.is_active:` for truth checks

Replace with `Project.is_active`

(E712)